src/api/app/views/webui2/webui/webui/_collapsible_text.html.haml

```diff
@@ -0,0 +1,8 @@
+- if text.present?
+  .obs-collapsible-textbox
+    - if text.length > threshold
+      .obs-collapsible-text.obs-collapsed
+        = simple_format(text, class: 'm-0 p-0')
+      %a.obs-collapsible-link.obs-collapsed.text-center.d-block{ title: 'Show more' }
```

Comment:
[nitpick]

It looks a bit weird with the arrow in the center in some cases, but I'm not sure if there is a solution for all the cases. Anyway, I tried it with `text-right` instead of `text-center`. This is the result:

![suggestion](https://user-images.githubusercontent.com/1102934/52948930-2d914580-337b-11e9-8603-8c2f1a6fa41d.png)

Reply:
I find it a bit to far away from the actual description in your screenshot.

Let's take it as is for now. We can still update this later on.